components/style/binding_tools/regen.py

```diff
@@ -291,7 +292,6 @@
             "nsIURI",
             "nsMainThreadPtrHolder",
             "nsRestyleHint",
-            "nsString",
```

Comment:
I got warnings about it not being used, after my first try doing a regen, so I thought I should!